.github/misc/FindTileDB_EP.cmake

```diff
@@ -0,0 +1,141 @@
+#
```

Comment:
Let's call this `TileDB-<version>-packages.cmake.in`, and use cmake configure file to do the substitution: https://cmake.org/cmake/help/latest/command/configure_file.html

Reply:
Also, it can live in https://github.com/TileDB-Inc/TileDB/tree/dev/cmake/inputs

Reply:
Hmm, I might not understand properly what you are trying to achieve. `configure_file(...)` can be called from within another CMake file, so we would need something like `generate.cmake` and `TileDB-<version>-packages.cmake.in` that would be used to generate `TileDB-<version>-packages.cmake`. In any case we still need to substitute all values in BASH script since it has the whole list-regex logic. Is this what you had in mind?:

- GitHub CI -> creates `generate.cmake` with all required values already in it.
- `generate.cmake` uses these values to configure_file `TileDB-.....cmake.in` (This could have been done by CI in the first step)
- upload the produced `TileDB-...cmake` with its SHA to release (We don't need to upload generate.cmake)

Do I understand it correctly?
If not could you perhaps explain your idea a bit more?